lib/rules/declaration-block-no-redundant-longhand-properties/index.mjs

```diff
@@ -275,11 +278,17 @@ const rule = (primary, secondaryOptions, context) => {
 					longhandDeclarationNode.push(decl);
 					longhandDeclarationNodes.set(prefixedShorthandProperty, longhandDeclarationNode);
 
-					const shorthandProps = longhandSubPropertiesOfShorthandProperties.get(shorthandProperty);
-					const prefixedShorthandData = Array.from(shorthandProps || [], (item) => prefix + item);
+					const shorthandProps = new Set(
+						longhandSubPropertiesOfShorthandProperties.get(shorthandProperty),
+					);
 
+					secondaryOptions?.ignoreLonghands?.forEach((/** @type {string} */ value) => {
+						if (shorthandProps.has(value)) shorthandProps.delete(value);
+					});
```

Comment:
[suggest] Extracting a local variable `ignoreLonghands`

```js
/** @type {string[]} */
const ignoreLonghands = secondaryOptions?.ignoreLonghands ?? [];
```

can simplify this code like this:

```suggestion
					ignoreLonghands.forEach((value) => shorthandProps.delete(value));
```

Just `.delete` seems enough (`.has` seems unneeded). Any thoughts?